emannotationschemas/__init__.py

```diff
@@ -6,18 +6,20 @@
 from emannotationschemas.postsynaptic_compartment import PostsynapticCompartment
 from emannotationschemas.base import FlatSegmentationReference
 from emannotationschemas.cell_type_local import CellTypeLocal
+from emannotationschemas.bound_text_tag import BoundTagAnnotation
+from emannotationschemas.extended_classical_cell_type import ExtendedClassicalCellType
 
 __version__ = '1.1.1'
 
 type_mapping = {
     'synapse': SynapseSchema,
     'presynaptic_bouton_type': PresynapticBoutonType,
     'postsynaptic_compartment': PostsynapticCompartment,
-    'cell_type_ai_manual': CellTypeLocal,
```

Comment:
we do have some annotations under this schema now.. i think its right to remove this but we might need to reinjest them